---
agency: Department of Health and Human Services
applicant_types:
- Government - General
- Interstate
- Intrastate
- Minority group
- Non-Government - General
- Other public institution/organization
- State
assistance_types:
- Direct Payments for a Specified Use
- Direct Payments with Unrestricted Use
- Dissemination of Technical Information
- Formula Grants
- Investigation of Complaints
- Project Grants
- Project Grants (with Formula Distribution)
- Training
- Use of Property, Facilities, or Equipment
authorizations:
- text: Public Health Service Act, Sections 301, 461 and 487, as amended; Public Laws
    78-410 and 99-158, 42 U.S.C. 241, as amended; 42 U.S.C. 285k; 42 U.S.C. 288; Small
    Business Research and Development Enhancement Act of 1992, Public Law 102-564.
  url: null
beneficiary_types:
- American Indian
- Asian
- Black
- Disabled (e.g. Deaf, Blind, Physically Disabled)
- Education (13+)
- Education Professional
- Engineer/Architect
- Federally Recognized Indian Tribal Governments
- Graduate Student
- Health Professional
- Local
- Minority group
- Native American Organizations
- Other Non-White
- Other private institution/organization
- Other public institution/organization
- Physically Afflicted (e.g. TB, Arthritis, Heart Disease)
- Private nonprofit institution/organization
- Profit organization
- Public nonprofit institution/organization
- Scientist/Researchers
- Small Business Person
- Small business
- Spanish Origin
- Sponsored organization
- State
- Student/Trainee
- U.S. Citizen
- U.S. Territories
- Women
categories:
- Health - Health Research - General
cfda: '93.172'
fiscal_year: '2022'
grants_url: https://grants.gov/search-grants?cfda=93.172
layout: program
objective: NHGRI supports the development of resources and technologies that will
  accelerate genome research and its application to human health and genomic medicine.
  A critical part of the NHGRI mission continues to be the study of the ethical, legal
  and social implications (ELSI) of genome research. NHGRI also supports the training
  and career development of investigators and the dissemination of genome information
  to the public and to health professionals.  The Small Business Innovation Research
  (SBIR) program is used to increase private sector commercialization of innovations
  derived from Federal research and development; to increase small business participation
  in Federal research and development; and to foster and encourage participation of
  socially and economically disadvantaged small business concerns and women-owned
  small business concerns in technological innovation. The Small Business Technology
  Transfer (STTR) program is used to foster scientific and technological innovation
  through cooperative research and development carried out between small business
  concerns and research institutions; to foster technology transfer between small
  business concerns and research institutions; to increase private sector commercialization
  of innovations derived from Federal research and development; and to foster and
  encourage participation of socially and economically disadvantaged small business
  concerns and women-owned small business concerns in technological innovation.
obligations: '[{"x":"2022","sam_estimate":0.0,"sam_actual":441171462.0,"usa_spending_actual":437149524.32},{"x":"2023","sam_estimate":459155961.0,"sam_actual":0.0,"usa_spending_actual":448406842.69},{"x":"2024","sam_estimate":456707217.0,"sam_actual":0.0,"usa_spending_actual":446986087.94}]'
permalink: /program/93.172.html
popular_name: Human Genome Project
results:
- description: "Encyclopedia of DNA Elements (ENCODE). After completing the full sequence\
    \ of the human genome, scientists faced the challenge of understanding what that\
    \ sequence means and how it contributes to health and disease. One approach NHGRI\
    \ has taken to address this question is to support the Encyclopedia of DNA Elements\
    \ (ENCODE) Project, which aims to identify the parts of the human genome sequence\
    \ that are functional, that is, sequences that are thought to play a critical\
    \ role in biological processes as measured by having some biochemical activity.\
    \ Research laboratories participating in the ENCODE Project use a variety of methods\
    \ to catalog the functional elements of the human genome. The resulting list of\
    \ functional elements, which includes genes and regions that control the expression\
    \ of genes, is presented as a resource that is freely available on the internet.\
    \ This resource gives scientists a new set of tools to use while investigating\
    \ biological phenomena and human disease.\r\n\r\n $30,000,000"
  year: '2016'
- description: 'ClinGen aims to build an authoritative central resource that defines
    the clinical relevance of genes and variants for use in precision medicine and
    research.  To do so, ClinGen investigators are developing standard approaches
    for sharing genomic and phenotypic data provided by clinicians, researchers, and
    patients through centralized databases, such as ClinVar, and are working to standardize
    the clinical annotation and interpretation of genomic variants.  Working groups
    are implementing evidence-based expert consensus methods to curate the clinical
    validity and medical actionability of genes and variants.   Experts in the areas
    of cardiovascular disease, pharmacogenomics, hereditary (germline) cancer, somatic
    cancer, and inborn errors of metabolism have been brought together to assist in
    these curation efforts.  ClinGen also aims to develop machine-learning algorithms
    to improve the throughput of variant interpretation and to improve understanding
    of variation in diverse populations as it relates to interpreting genetic test
    results.  Lastly, ClinGen will disseminate the collective knowledge and resources
    for unrestricted use in the community and for use in EHR ecosystems. '
  year: '2017'
- description: "The ELSI Research Program funds research studies, training opportunities\
    \ and workshops, and develops and supports research consortia and conferences\
    \ in the following broad areas. \r\n\r\nGenomic Research - Projects in this area\
    \ examine and address the issues that arise in the design and conduct of genomic\
    \ research, particularly as it involves the production, analysis and broad sharing\
    \ of individual genomic data that is frequently coupled with detailed health information.\
    \  \r\nGenomic Health Care - Projects in this area explore how rapid advances\
    \ in genomic technologies and the availability of increasing amounts of genomic\
    \ information influence how health care is provided and how it affects the health\
    \ of individuals, families and communities. \r\nBroader Societal Issues - Projects\
    \ in this area examine the normative underpinnings of beliefs, practices and policies\
    \ regarding genomic information and technologies, as well as the implications\
    \ of genomics for how we conceptualize and understand such concepts as health,\
    \ disease, and individual responsibility.\r\nLegal, Regulatory and Public Policy\
    \ Issues - Projects in this area explore the effects of existing genomic research,\
    \ health and public policies and regulations and provide data to inform the development\
    \ of new policies and regulatory approaches.\r\n\r\nA more detailed description\
    \ of these areas and a list of examples of possible research questions related\
    \ to each are available on the ELSI Research Priorities website: http://www.genome.gov/27543732.\r\
    \n"
  year: '2018'
- description: 'The mission of the ENCODE Project is to enable scientific and medical
    communities to interpret the human genome sequence to better understand human
    biology and to improve health. ENCODE seeks to achieve that goal through several
    means, one of which is through data production. ENCODE data are rapidly released
    to the research community after undergoing rigorous quality control analysis.
    Data are compiled into an “encyclopedia” (see: www.encodeproject.org/data/annotations/)
    to provide genome annotations that can be accessed by a wide range of users. The
    ENCODE Consortium is an open project that includes investigators with diverse
    scientific backgrounds and expertise in the production and analysis of data. In
    addition to the production of nearly 900 ENCODE Consortium publications, there
    are now more than 2200 scientific publications from groups without ENCODE funding
    who have used ENCODE data for their published work (community publications). The
    widespread use of ENCODE data is fostered by outreach and collaboration efforts.
    Efforts to annotate both protein-coding and non-coding regions of the human genome
    have been implemented by the ENCODE Project. Genome-wide association studies (GWAS)
    studies have shown that most disease-related variants reside in non-coding regions
    and that most of the heritability of common diseases has been linked to non-coding
    regions. The potential influence of non-coding regions in disease highlights the
    importance of the ENCODE Project’s genome annotations. Throughout the project,
    standards to ensure high-quality data have been implemented and novel algorithms
    have been developed to facilitate analysis. Data and derived results are made
    available as a community resource through a freely accessible database (see: www.encodeproject.org/),
    enabling broad use of the ENCODE catalog of functional elements by experimental
    and computational biologists.

    The ENCODE Project seeks to identify all functional'
  year: '2019'
- description: "Clinical Sequencing Evidence-Generating Research (CSER) Program\n\
    A key feature of CSER was integrated and coordinated efforts to address challenges\
    \ and opportunities in targeted areas, implemented largely through Working Groups:\
    \ Actionability-Return of Results, Electronic Health Records, Genetic Counseling,\
    \ Informed Consent and Governance, Outcomes and Measures, Pediatrics, Practitioner\
    \ Education, Sequencing Standards, and Tumor. At individual sites and across CSER,\
    \ investigators have produced over 340 papers describing challenges and opportunities\
    \ across the spectrum of clinical sequencing and its implementation. Findings\
    \ are also disseminated through talks and plenary sessions at major national meetings,\
    \ such as the ASHG, ACMG, NSGC, ASCO, AACR, and ASBH annual meetings. These efforts\
    \ have culminated in numerous scientific advances, including models for genomics-oriented\
    \ informed consent tailored to the care setting,\nrecommendations to improve the\
    \ consistency of genomic variant interpretation, and approaches to the disclosure\
    \ of primary pediatric and tumor findings and of secondary findings more broadly.\
    \ Collaborating with the Electronic Medical Records and Genomics (eMERGE) network,\
    \ CSER identified barriers and recommended approaches to incorporating genomic\
    \ information in electronic health records. Valuable CSER products also included\
    \ contributions to and leadership on three sets of ACMG recommendations (relating\
    \ to secondary findings, variant interpretation, and clinical laboratory standards),\
    \ case studies and single site publications, methodological and tool development,\
    \ and collaborations with other consortia. CSER research products are available\
    \ at: https://cser-consortium.org/cser-research-materials.\nThe second phase of\
    \ CSER is building on work produced in the first phase to rigorously assess the\
    \ clinical utility of genome sequencing and to support the integration of genomic,\
    \ clinical, and healthcare utilization data in real-world healthcare systems to\
    \ inform clinical decision making. In February 2017, NHGRI Council reviewed a\
    \ funding plan for 6 CSER sites to be funded under RFA-HG-16-011, and a Coordinating\
    \ Center to be funded under RFA-HG-16-012, and awards were made in August 2017.\
    \ Across the CSER sites, approximately 8,000 participants, at least 60% of whom\
    \ will be from racial or ethnic minority or medically underserved populations,\
    \ will be recruited. Crucial and complementary to these efforts is a continuing\
    \ focus on ELSI research across the aims of each funded project and a dedicated\
    \ focus on engaging stakeholders such as patients and parents of pediatric patients,\
    \ clinicians, community members, patient advocates, health system leadership,\
    \ and payers.\nThe CSER sites and their goals are described further in the September\
    \ 2018 marker paper (Amendola, et al. AJHG, PMID 30193136). Goals of the consortium\
    \ include assessing the clinical utility of genomic sequencing, exploring medical\
    \ follow up and cascade testing of relatives, and evaluating patient-provider-laboratory\
    \ level interactions that influence the use of this technology. Five of the 6\
    \ extramural sites are focused on pediatric populations. Recruitment is underway\
    \ at all clinical sites, and 8 Working Groups are developing ideas for manuscripts\
    \ to leverage cross-CSER data. Having just begun its fourth project year, CSER\
    \ has developed a set of harmonized baseline and follow-up measures for study\
    \ participants and providers, a decliner survey, and a survey of health system\
    \ experts (https://cser-consortium.org/cser-research-materials). These measures\
    \ were chosen to align with a comprehensive framework for genomic medicine integrative\
    \ research (Horowitz, et al. AJHG, PMID 31104772). Bruce Korf (UAB/SouthSeq study)\
    \ created a CSER schema that outlines the workflow and major research questions\
    \ surrounding the clinical application of genome sequencing. This schema is being\
    \ used to categorize CSER publications according to which part of the workflow\
    \ they address.\nParticipant recruitment and follow-up, establishment of a consortium-wide\
    \ data sharing platform, and data collection for CSER-wide manuscripts are in\
    \ process. Ultimately, the findings from the CSER consortium will offer patients,\
    \ healthcare systems, and policymakers a clearer understanding of the opportunities\
    \ and challenges of providing genomic medicine in diverse populations and settings,\
    \ and contribute evidence toward developing best practices for the delivery of\
    \ clinically useful and cost-effective genomic sequencing in diverse healthcare\
    \ settings.\nThe NHGRi Genomic Data Science Analysis, Visualization, and Informatics\
    \ Lab-Space (AnVIL)\nSince its soft launch in June 2019, AnVIL has publicly released\
    \ high-value datasets, such as GTEx version 8 and the high-coverage 1000 Genomes\
    \ data, and currently holds data from the NHGRI Genome Sequencing Program (GSP)\
    \ and the Electronic Medical Records and Genomics (eMERGE) for access by GSP consortium\
    \ members. Key analysis tools and packages such as Jupyter Notebooks, RStudio,\
    \ and Bioconductor have also been made available in the users’ workspaces.  To\
    \ get objective advice on technical achievements and future milestones, the AnVIL\
    \ team has convened two meetings with the project’s External Consultant Committee\
    \ (ECC). \n \nAs a training event for researchers of the Centers for Common Disease\
    \ Genomics (CCDGs) and Centers for Mendelian Genomics (CMGs) programs, the AnVIL\
    \ Outreach working group held the Massive Genome Informatics in the Cloud (MaGIC)\
    \ Jamboree on June 10 – 11, 2020—an event to introduce consortium researchers\
    \ to the available data, tools, workflows, training materials, and support channels\
    \ on AnVIL. \n \nAnVIL is also a component of the emerging NIH federated data\
    \ resource ecosystem and is expected to collaborate and integrate with other genomic\
    \ data resources through the adoption of the FAIR (Findable, Accessible, Interoperable,\
    \ Reusable) principles, as their specifications emerge from the genomics community.\
    \ AnVIL program staff have a major role in current trans-NIH activities to facilitate\
    \ interoperability and maintain communication among NIH cloud-based genomic resources,\
    \ such as the NCI Cancer Research Data Commons, the NHLBI BioData Catalyst, and\
    \ the Gabriella Miller Kids First Pediatric Research Program. The AnVIL program\
    \ team leads and coordinates the cross-platform interoperability projects and\
    \ workshops with key personnel of these resources, under the project name NIH\
    \ Cloud Platforms Interoperability (NCPI). Within one year since the project began,\
    \ the AnVIL team has hosted a virtual, project-wide conference for the NCPI working\
    \ groups to share their progress and collaborate. Additionally, the AnVIL project\
    \ team led a “Train Your Colleague” workshop, a cross-training event where users\
    \ and developers of the NCPI project learned about each of the four platforms."
  year: '2021'
- description: "The AnVIL Program serves as an interoperable resource for the research\
    \ community by co-locating data, storage, and computing infrastructure with commonly\
    \ used services and tools for analyzing and sharing data.\n\nSince its launch\
    \ in June 2019, the AnVIL Program has onboarded and publicly released several\
    \ high-value datasets (see the AnVIL Dataset Catalog), and deployed key genomic\
    \ analysis tools and packages such as Jupyter Notebooks, RStudio, Galaxy, Dockstore,\
    \ Bioconductor, and PharmCAT. The AnVIL platform was also integral to the analysis\
    \ of the first complete human genome of the Telomere-to-Telomere Consortium effort.\
    \   A summary of the AnVIL Program's accomplishments includes: \n\nthe ingestion\
    \ of over 600,000 samples (4.56 Pb of data) into AnVIL  \n\nthe deployment of\
    \ Terra, for workspaces, interactive and batch computing \n\nthe deployment of\
    \ Gen3 within AnVIL \n\nrelease of Galaxy within AnVIL/Terra \n\nenhanced capabilities\
    \ for Dockstore \n\nenhanced capabilities for Jupyter notebooks \n\ndeployment\
    \ of RStudio available within Terra \n\nthe development and deployment of the\
    \ Bioconductor AnVIL packages \n\nestablishing the AnVIL Portal as a meta-portal\
    \ to each of the AnVIL components \n\nseveral successful outreach events \n\n\
    launched two cohorts of the AnVIL Cloud Credits Program \n\nlaunched a new Genomic\
    \ Data Science Community Network \n\ndeveloping an initial catalog of clinical\
    \ genomics tools \n\nlaunch of major efforts through the NIH Cloud Platform Interoperability\
    \ (NCPI) program to increase usability across NIH funded cloud platforms \n\n\
    the piloting of the Data Use Oversight System (DUOS) \n\negress-free Release of\
    \ the GTEx V8 Protected Data \n\nThe AnVIL team continues to engage the genomics\
    \ research communities by presenting posters and holding workshops. For example,\
    \ the AnVIL team has presented at the Bioinformatics Community Conference, the\
    \ American Society of Human Genetics meeting, and has hosted a webinar at the\
    \ 2021 Bioconductor Virtual Conference. In 2021 the AnVIL team launched the AnVIL\
    \ Cloud Credits Program (AC2) which invited genomic researchers to submit proposals\
    \ that use the AnVIL platform for large-scale data analysis with cloud computing\
    \ credits supported by the NIH STRIDES program.  Six investigators received awards\
    \ from the AC2 program to use AnVIL as their analysis platform. After the success\
    \ of AC2, the program was expanded in 2022 with 13 additional cloud credit awardees.\
    \  \n\nThe AnVIL Program is also committed to addressing disparities in equitable\
    \ access to resources for performing large scale genomic based analysis. In March\
    \ 2021, the AnVIL team helped organize the first meeting of the Genomic Data Science\
    \ Community Data Network (GDSCN). This meeting brought together over 40 researchers\
    \ from a diverse spectrum of institutions. The focus of the GDSCN is to build\
    \ partnerships and develop genomic data science curricula for undergraduate students.\
    \ The AnVIL team also partnered with the NIH Office of Data Science Strategy and\
    \ Howard University’s Virtual Applied Data Science Training Institute (VADSTI)\
    \ to develop a free eight-week virtual training series. VADSTI aims to attract\
    \ and engage underrepresented students and researchers interested in applying\
    \ data science applications to biomedical, clinical, and genomic research, with\
    \ a focus on diseases common to minority populations.  The training sessions covered\
    \ foundational analytic skills needed for research using big data and included\
    \ an AnVIL team led session on how to use the cloud to make biomedical data science\
    \ discoveries. \n\n The AnVIL Program is a key component of the emerging ecosystem\
    \ of data resources at NIH and is expected to collaborate and integrate with other\
    \ genomic data resources through the adoption of the FAIR (Findable, Accessible,\
    \ Interoperable, Reusable) principles, as their specifications emerge from the\
    \ genomics community. The AnVIL team has collaborated with the Researcher Auth\
    \ Service (RAS) Initiative to facilitate access to NIH’s controlled data assets\
    \ and repositories in a consistent manner.  \n\nThe AnVIL Program is committed\
    \ to contributing to and ensuring interoperability with Fast Healthcare Interoperability\
    \ Resources (FHIR) to help address the needs of the clinical genomic research\
    \ community. The FHIR standard describes data formats and elements (known as \"\
    resources\") and an API for exchanging electronic health records (EHR). In support\
    \ of this need, Bioconductor and Galaxy have leveraged FHIR for research purposes.\
    \ Furthermore, the open-source AnVIL-FHIR python package (pyAnVIL) was initiated\
    \ on github, and over the course of the past year, has been substantially expanded\
    \ and enhanced."
  year: '2023'
sam_url: https://sam.gov/fal/171c0fc7f25f40ee88b042690e9db4dc/view
sub-agency: National Institutes of Health
title: Human Genome Research
usaspending_url: https://www.usaspending.gov/search/?hash=e11cbfead323459494e0d64d0e8ac09b
---
